informer/informer.go

```diff
@@ -57,20 +53,6 @@ type Config struct {
 	ResyncPeriod time.Duration
 }
 
-// DefaultConfig provides a default configuration to create a new by best
-// effort.
-func DefaultConfig() Config {
-	return Config{
-		// Dependencies.
-		Watcher: nil,
-
-		// Settings.
-		ListOptions:  metav1.ListOptions{},
-		RateWait:     DefaultRateWait,
```

Comment:
I think the rate wait can actually be zero, but then we do not know how to default it to something else. Right now the default is one second and I would like to keep this but the defaulting gets broken like with booleans now. Anybody a cool idea how to do this? I only have using pointers in mind.

Reply:
Why not default to DefaultRateWait like it was?

Reply:
The thing is you can just configure 0. This means there is then no rate wait. This means 0 is not the default. Based on what should we default? We only get input the user manipulated or not. We do not know if they did so. This is why the only reliable option I have in mind are pointers.

Reply:
Ack.